dialogflow/stopwatch/README.md

```diff
@@ -1,42 +1,65 @@
 # Dialogflow Sample
 
-This app demonstrates how to make streaming gRPC connections to the [Dialogflow API](https://cloud.google.com/dialogflow-enterprise/) to recognize commands in recorded audio.
+This app demonstrates how to make streaming gRPC connections to the [Dialogflow API]
+
+The app demonstrates how to detect intents:
+- Via Text
+- Via Streaming Audio
+- With Sentiment Analysis
+- With Text-to-Speech
+- With Knowledge Connectors
+
+To call the Dialogflow API from iOS, you need to provide authorization tokens with each request for them to be accepted by the Dialogflow API. To get this token, this sample uses a Firebase Function (in Node.js) to generate these tokens on the behalf of a service account to be used by the app when making a request to the Dialogflow API.
 
 ## Prerequisites
-- Credentials for calling the Dialogflow API
-- An OSX machine or emulator
+- An OSX machine or simulator
 - [Xcode 9.1][xcode] or later
 - [Cocoapods][cocoapods] version 1.0 or later
 
-## Prepare your Google Cloud account
-As with all Google Cloud APIs, every call to the Dialogflow API must be associated
-  with a project within the [Google Cloud Console][cloud-console] that has the
-  API enabled. In brief:
-  - Create a project (or use an existing one) in the [Cloud Console][cloud-console]
-  - [Enable billing][billing].
+## Setup
+- Create a project with the [Google Cloud Console][cloud-console]
+- Enable the [Dialogflow API](https://console.cloud.google.com/apis/librarydialogflow.googleapis.com).
+- [Enable billing][billing].
+- [Import the Dialogflow Agent](https://dialogflow.com/docs/agents/export-import-restore#import) using file xyz
```

Comment:
Should provide the agent file.

Reply:
Provided the agent `StopwatchAgent.zip` under the `Stopwatch` directory.